pbs2slurm.py

```diff
@@ -249,8 +248,26 @@ def _repl(m):
                 elif len(s) > 2:
                     return ""
                 return "#SBATCH --time={}:{}:{}".format(h, m, s)
+            elif "nodes" in resources:
+                mydir = ''
+                rlist = resources.split(':')
+                for rx in rlist:
+                   cx = rx.split(',')
+                   for nx in cx:
+                       mx = nx.split('=')
+                       if mx[0] == 'nodes':
+                           mydir = "#SBATCH --ntasks={}".format(mx[1].strip())
+                       elif mx[0] == 'ppn':
+                           mydir = mydir + "\n#SBATCH --cpus-per-task={}".format(mx[1].strip())
+                       elif mx[0] == 'pmem': 
+                           mydir = mydir + "\n#SBATCH --mem-per-cpu={}".format(mx[1].strip())
```

Comment:
I'm not sure about the mapping of `-l nodes` to `--ntasks`. It has been a while since i looked at this or PBS for that matter but i thought that
```
#PBS -l nodes=2:ppn=4
```
translated into 8 MPI ranks. It seems to me that this should translate to
```
#SBATCH --nodes=2
#SBATCH --ntasks-per-node=4
```
(ignoring the question of `--ntasks-per-core`)

However, the distinct use of `--ntasks` for distributed parallelism and `--cpus-per-task` for multithreading or simple multiprocessing makes this more complicated. Once could assume that for single node jobs like
```
#PBS -l nodes=1:ppn=12
```
it could translate to
```
#SBATCH --ntasks=1
#SBATCH --cpus-per-task=12
```

I have to think about this some more